src/main/ui/app/component/NamedList.jsx

```diff
@@ -27,9 +27,8 @@ export default class NamedList extends Component {
         const COLUMNS_IF_SIZE = 5;
 
         const list = this.props.list;
-        const colSize = list.addOns.length > COLUMNS_IF_SIZE ?
```

Comment:
Ideally we'd preserve the behavior of using two columns to show more items on larger screens.

So, first I'd like to try bringing back that behavior. (But if we're going to remove it, then we should also move the COLUMNS_IF_SIZE const above.